lib/s3ff/view_helper.rb

```diff
@@ -32,24 +32,31 @@ def include_s3ff_templates
                 <button class="close" data-dismiss="alert" data-unique_id="{{:unique_id}}" data-target="#upload-{{:unique_id}}" style="margin-left:1em;" type="button">
                   <span aria-hidden="true">&times;</span><span class="sr-only">Remove</span>
                 </button>
-                <span class="fa fa-file-o"></span>
-                {^{:result.filename}}
+                <span class="filename">
```

Comment:
Wrap the `file icon` and the `filename` in another `span`. This provides the option of hiding the `file icon` and `filename` without hiding the `close button`.